src/renderer/views/account/CreateAccountView.tsx

```diff
@@ -34,11 +34,15 @@ const CreateAccountView = observer(
         password
       );
 
-      const privateKey: PrivateKey = ipcRenderer.sendSync(
-        "unprotect-private-key",
-        address,
-        password
-      );
+      const [privateKey, passphraseOrError]: [
```

Comment:
It seems the second element of the return value has been never `passphrase`.

https://github.com/planetarium/9c-launcher/blob/78c0ee02eca6a18df6af74d1dcee1b0cf3979bb4/src/main/main.ts#L539-L562

So it should rename into `error`.

Reply:
Oh my mistake. The js(ts) convention is confusing to me.😢

Reply:
The second element can be `{}`, `undefined`, `error: any`. How can I naming it? Hm... Just `any` or `_` I think.

Reply:
Here is [a interesting article](https://ntgard.medium.com/tuples-in-javascript-cd33321e5277)
```
const [, , thisOne, , thatOne] = getTuple();
const [_1, _2, thisOne, _3, thatOne] = getTuple();
```

Reply:
Then, I suggest to ignore the second element like the post you found `const [privateKey] = ipc...` if you will check the error with only `privateKey` state without `error`. But I suggest also to check with `error`.

And I have a question, what the case the second element became `{}`?

Reply:
The second element became `{}` on line 547.

Reply:
I see 👍🏻

Reply:
The `{}` on line 547 can change to `error: string` that contains error message. How's it? @moreal 
And store the second element as `error: any | undefined` type and handles the error.

Reply:
> And store the second element as `error: any | undefined` type and handles the error.

`any` contains `undefined` so there is no reason to use `any | undefined`.

> The `{}` on line 547 can change to `error: string` that contains error message

Could you describe more about the change you mean? If you mean making it return other meaningful value instead `{}`, it seems better to return `Error` instead `string`, to suit for `Error` concepts.